app/src/main/java/com/example/soundcloud/selected_genre_detail/SelectedGenreActivity.java

```diff
@@ -16,6 +16,8 @@
     private TextView mTextViewGenreTitle;
     private ImageButton mImageButtonSearch;
```

Comment:
ImageButton cũng có thể đặt là Button
```suggestion
    private ImageButton mButtonSearch;
```